partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnTrimTrailing.kt

```diff
@@ -3,13 +3,13 @@
 
 package org.partiql.spi.function.builtins
 
-import org.partiql.spi.function.Function
 import org.partiql.spi.function.Parameter
+import org.partiql.spi.function.utils.FunctionUtils
 import org.partiql.spi.function.utils.StringUtils.codepointTrimTrailing
 import org.partiql.spi.value.Datum
 import org.partiql.types.PType
 
-internal val Fn_TRIM_TRAILING__STRING__STRING = Function.static(
```

Comment:
Seems like some other functions aren't hidden yet. Taking a look at the https://github.com/johnedquinn/partiql-lang-jvm/blob/main-builtins-names/partiql-spi/src/main/kotlin/org/partiql/spi/function/Builtins.kt#L21,
- abs
- bit_length
- cardinality
- char_length
- lower
- (some others)

Reply:
These are public in SQL. See Section 20.70 and 6.17.

Reply:
I see. Perhaps could be good to add a comment at some place (perhaps in the FunctionUtils.kt) that these SQL99-defined scalar and aggregation function names are purposely not hidden.